website/blog/2017-09-15-1.7.0.md

```diff
@@ -0,0 +1,219 @@
+---
+author: @suchipi
+authorURL: https://twitter.com/suchipi
+title: "Prettier 1.7: JSX tweaks, Pragma, TypeScript and CSS fixes"
+---
+
+This release features some bugfixes and tweaks around JSX, TypeScript, CSS, and JavaScript formatting, as well as a couple new features.
```

Comment:
I like how this blog post has a one-sentence description of what changed. Could you add something like this to the older blog posts, followed by the `<!-- truncate -->`, so that people don’t have to scroll through pages and pages of text on the blog homepage?

Reply:
Done!